reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-61.md

```diff
@@ -37,6 +37,12 @@ so that any modules or scripts that use these APIs can rely on them being availa
 
 The Windows Compatibility Pack enables PowerShell Core to use **more than 1900 cmdlets that ship with Windows 10 October 2018 Update and Windows Server 2019**.
 
+## Support for Application Whitelisting
+
+PowerShell Core 6.1 has parity with Windows PowerShell 5.1 supporting [AppLocker](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-application-control/applocker/applocker-overview)
```

Comment:
"PowerShell Core 6.1 has parity with Windows PowerShell 5.1"
Actually this isn't quite true.  PSCore will not have parity with Windows PowerShell until the merge work is completed.  I think it is more accurate to say that this release of PSCore has added application white listing support for Windows platform.

Reply:
I distinctly called out feature parity (specific to Lee's feature table).